StepmaniaCore.cmake

```diff
@@ -48,28 +48,51 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 # Checks the standard include directories for c-style headers.
 # We may use C++ in this project, but the check works better with plain C headers.
 include(CheckIncludeFiles)
-check_include_files(math.h HAVE_MATH_H)
 check_include_files(alloca.h HAVE_ALLOCA_H)
+check_include_files(assert.h HAVE_ASSERT_H)
```

Comment:
There is already a dedicated place to check for different files and symbols. They were moved here since other projects may need them.